### ISSUE TITLE
Found markup element with unexpected name 'PageTitle' in .NET Blazor app in ubuntu 24.04

### ISSUE THREAD

**Issue by SoumyadipYT-OSS** [dotnet/website-feedback#238]
### URL

https://dotnet.microsoft.com/en-us/learn/aspnet/blazor-tutorial/run

### Operating system

linux

### More information about your system

My system is Ubuntu 24.04.1 LTS, and it is x86 architecture based. OS version: x11

### Description

The app is working properly when using 'dotnet watch' in vs code, but in every pages directory, let's say 'Home.razor' the default code is:

```razor
@page "/"

<PageTitle>Home</PageTitle>

<h1>Hello, world!</h1>

Welcome to <b>.NET Blazor</b> app.
```

here the warning underline is shown on the `<PageTitle>`, the warning is showing following below I've copy pasted from the IDE:

Found markup element with unexpected name 'PageTitle'. If this is intended to be a component, add a @using directive for its namespace.Razor(RZ10012)

### dotnet --info

smajumder@Lenovo:~$ dotnet --info
.NET SDK:
 Version:           8.0.108
 Commit:            665a05cea7
 Workload version:  8.0.401-baseline.24412.2

Runtime Environment:
 OS Name:     ubuntu
 OS Version:  24.04
 OS Platform: Linux
 RID:         ubuntu.24.04-x64
 Base Path:   /usr/lib/dotnet/sdk/8.0.108/

.NET workloads installed:
 Workload version: 8.0.401-baseline.24412.2
 [android]
   Installation Source: SDK 8.0.100
   Manifest Version:    34.0.43/8.0.100
   Manifest Path:       /home/smajumder/.dotnet/sdk-manifests/8.0.100/microsoft.net.sdk.android/34.0.43/WorkloadManifest.json
   Install Type:        FileBased

 [macos]
   Installation Source: SDK 8.0.100
   Manifest Version:    14.0.8478/8.0.100
   Manifest Path:       /home/smajumder/.dotnet/sdk-manifests/8.0.100/microsoft.net.sdk.macos/14.0.8478/WorkloadManifest.json
   Install Type:        FileBased

 [maui-windows]
   Installation Source: SDK 8.0.100
   Manifest Version:    8.0.3/8.0.100
   Manifest Path:       /home/smajumder/.dotnet/sdk-manifests/8.0.100/microsoft.net.sdk.maui/8.0.3/WorkloadManifest.json
   Install Type:        FileBased

 [maui-android]
   Installation Source: SDK 8.0.100
   Manifest Version:    8.0.3/8.0.100
   Manifest Path:       /home/smajumder/.dotnet/sdk-manifests/8.0.100/microsoft.net.sdk.maui/8.0.3/WorkloadManifest.json
   Install Type:        FileBased

Host:
  Version:      8.0.8
  Architecture: x64
  Commit:       08338fcaa5

.NET SDKs installed:
  8.0.108 [/usr/lib/dotnet/sdk]

.NET runtimes installed:
  Microsoft.AspNetCore.App 8.0.8 [/usr/lib/dotnet/shared/Microsoft.AspNetCore.App]
  Microsoft.NETCore.App 8.0.8 [/usr/lib/dotnet/shared/Microsoft.NETCore.App]

Other architectures found:
  None

Environment variables:
  DOTNET_ROOT       [/home/smajumder/.dotnet]

global.json file:
  Not found

Learn more:
  https://aka.ms/dotnet/info

Download .NET:
  https://aka.ms/dotnet/download

smajumder@Lenovo:~$ 
smajumder@Lenovo:~$

**Comment by katiesavage** [comment]
Did you manage to run the app on the previous step and you started getting this error on this step only? And are you using the CLI for the tutorial?

cc @danroth27

**Comment by danroth27** [comment]
@SoumyadipYT-OSS It sounds like you're hitting an issue where the `PageTitle` component isn't getting recognized in the editor. Improving the reliability of the Razor editor is something we're actively working on. Can you try closing and reopening VSCode and see if that resolves the issue.

@phil-allen-msft @leslierichardson95 FYI.

**Comment by SoumyadipYT-OSS** [comment]
@danroth27 Yes the issue solved, first the primary root cause was that there were not two workloads there for ubuntu 24.04, after removing the iOS and tizen from the workloads that is <TargetFramework> </TargetFramework> part, the issue solved.

**Comment by mairaw** [comment]
is that expected @danroth27?

**Comment by danroth27** [comment]
> is that expected [@danroth27](https://github.com/danroth27)?

@mairaw No, I don't know of any reason why uninstalling the iOS or Tizen workloads should impact the Razor tooling. I suspect its unrelated and coincidental.

**Comment by danroth27** [comment]
@SoumyadipYT-OSS Which editor where you seeing the reported error in? Was this in Visual Studio Code with the C# extension installed?

**Comment by SoumyadipYT-OSS** [comment]
@danroth27 I was using VsCode in my Ubuntu OS (24.04LTS) with .NET 8.

**Comment by danroth27** [comment]
@phil-allen-msft Do you want this issue moved to the dotnet/razor repo? Or should close it as duplicate against some existing issue?

**Comment by danroth27** [comment]
I'll go ahead and transfer it. We can the dup it as part of normal triage.
